readme.txt

```diff
@@ -1,8 +1,4 @@
-When generating the OBIEE jax-ws classes, maven needs to talk directly to the server; it can't use the proxy.  I don't know why this is.
+# ws-obiee-client, aka Rowmap
```

Comment:
Should this be `obiee-ws-client` instead?

Reply:
Doh!

Reply:
fixed